src/app/pages/app-bar/dropdown-toolbar/dropdown-toolbar.component.html

```diff
@@ -0,0 +1,17 @@
+<pxb-dropdown-toolbar #toolbar title="Title" [subtitle]="getSelected() || 'subtitle'">
```

Comment:
may want to give it a z4 dropdown shadow

Reply:
You got it.